sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -773,18 +871,22 @@ export interface CreateOperationInput {
 
 // @public
 export class Database {
-    constructor(client: CosmosClient, id: string, clientContext: ClientContext);
+    constructor(client: CosmosClient, id: string, clientContext: ClientContext, encryptionManager?: EncryptionManager, _rid?: string);
```

Comment:
This constructor needs to be marked hidden/internal. Users cannot instantiate this class as clientContext is internal class.

Reply:
done